lib/data_migrate/data_schema.rb

```diff
@@ -30,10 +30,12 @@ def migrated
 
     def versions
       @versions ||= begin
-        versions = []
-        Dir.foreach(DataMigrate::DataMigrator.full_migrations_path) do |file|
-          match_data = DataMigrate::DataMigrator.match(file)
-          versions << match_data[1].to_i if match_data
+        versions = Set.new
+        DataMigrate::DataMigrator.migrations_paths.each do |path|
+          Dir.foreach(path) do |file|
+            match_data = DataMigrate::DataMigrator.match(file)
+            versions << match_data[1].to_i if match_data
+          end
         end
         versions
```

Comment:
`Set` and `Array` have pretty different methods available. If it doesn't break anything, I think changing to `Set` here is an improvement since it removes duplicates. But if we wanted to minimize changes to method signatures, we could put a `.to_a` here so the method returns the same type as it did before.